src/main/scala/li/cil/oc/integration/projectred/EventHandlerProjectRed.scala

```diff
@@ -6,10 +6,11 @@ import net.minecraft.item.ItemStack
 
 object EventHandlerProjectRed {
   def useWrench(player: EntityPlayer, x: Int, y: Int, z: Int, changeDurability: Boolean): Boolean = {
```

Comment:
Shouldn't the x, y, and z parameters be replaced with a BlockPos parameter? Other integrations do that.